content/js/users.md

```diff
@@ -108,6 +108,19 @@ const handler = skygear.onUserChanged(function (user) {
 handler.cancel(); // The callback is cancelable
 ```
 
+### Who am I
```

Comment:
@ben181231 I think we need to highlight the difference between `whoami` and `skygear.currentUser`.